DNN Platform/Modules/MemberDirectory/Services/MemberDirectoryController.cs

```diff
@@ -169,7 +169,7 @@ private IEnumerable<UserInfo> GetUsers(int userId, int groupId, string searchTer
                                                                            sortField, (sortOrder == "ASC"),
                                                                            "DisplayName", searchTerm)
                                                                            :
-                                                                           UserController.Instance.GetUsersAdvancedSearch(portalId, PortalSettings.UserId, -1, -1,
+                                                                           UserController.Instance.GetUsersAdvancedSearch(portalId, PortalSettings.UserId, userId, -1,
```

Comment:
The filterUserId parameter only used when search type is "Relationship", so IMO it can keep use -1 here.